homeassistant/components/demo/strings.json

```
@@ -61,5 +61,17 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "mode": {
```

Just so the demo doesn't cause confusing, maybe make use a more specific key than `mode`?
```suggestion
      "hvac_setting": {
```

---

will adjust 👍